prow/README.md

```diff
@@ -442,6 +442,18 @@ You may also have to create a keypair with the Metal3 CI ssh key.
     kubectl apply -k manifests/overlays/metal3
    ```
 
+1. Create config (updated automatically by prow after this)
+
+   ```bash
+   kubectl -n prow create configmap config --from-file=config/config.yaml
+   kubectl -n prow create configmap plugins --from-file=config/plugins.yaml
+   kubectl -n prow create configmap label-config --from-file=config/labels.yaml
+   kubectl -n prow create configmap job-config \
+      --from-file=config/jobs/metal3-io \
+      --from-file=config/jobs/Nordix \
+      --from-file=config/jobs
```

Comment:
This is a bit error prone, since we need to remember to add new folders. On the other hand, prow will autoupdate the configmap as long as the config updater is there in the `plugins.yaml`. This means that as long as we create *some* configmap here that prow can mount, it will happily sync with folders we forgot on the first reconcile.